research/minigo/minigo.py

```diff
@@ -259,20 +244,21 @@ def validate(trained_models_dir, holdout_dir, estimator_model_dir, params):
     dualnet.validate(estimator_model_dir, tf_records, params)
 
 
-def evaluate(trained_models_dir, black_model_name, white_model_name,
+def evaluate(black_model_name, black_net, white_model_name, white_net,
              evaluate_dir, params):
   """Evaluate with two models.
 
-  With the model name, construct two DualNetRunners to play as black and white
-  in a Go match. Two models play several names, and the model that wins by a
-  margin of 55% will be the winner.
+  With two DualNetRunners to play as black and white in a Go match. Two models
+  play several names, and the model that wins by a margin of 55% will be the
+  winner.
 
   Args:
-    trained_models_dir: Directories where the completed generations/models are.
     black_model_name: The name of the model playing black.
+    black_net: The actual DualNetRunner model for black
```

Comment:
nit: "actual" here makes sense when you know it used to be a name, but doesn't make sense if you read this for the first time. That word can be removed.